pages/common/sonar-scanner.md

```diff
@@ -1,6 +1,6 @@
 # sonar-scanner
 
-> SonarScanner is a generic scanner for SonarQube for projects do not use any specific build tool like maven, gradle , etc.
+> SonarScanner is a generic scanner for SonarQube for projects do not use any specific build tool like maven, gradle, etc.
```

Comment:
```suggestion
> SonarScanner is a generic scanner for SonarQube for projects do not use any specific build tool like Maven, Gradle, etc.
```

Reply:
Is it me... or this sentence does not make any sense? :innocent:

Reply:
```suggestion
> SonarScanner is a generic scanner for SonarQube for projects, which do not use any specific build tool like maven, gradle, etc.
```
Still not happy with the `for SonarQube for projects`, but at least it's a bit better now

Reply:
You guys are both totally right — I didn't even pay a speck of attension to the rest of the sentence until _just_ now 😄

Reply:
This is much better, right?
```suggestion
> SonarScanner is a generic scanner for SonarQube projects that do not use build tools such as Maven, Gradle, or Ant.
```

Reply:
Hrm, this is an awkward one - I completely agree on the double `for` there. What about this?
```suggestion
> SonarScanner is a generic SonarQube scanner for projects that do not use any specific build tool such as Maven or gradle.
```

Ref https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/ for additional docs on the thing I think

Reply:
`gradle`-> `Gradle` 😇